src/Wallabag/CoreBundle/Helper/DownloadImages.php

```diff
@@ -193,9 +193,11 @@ protected function getSrcsetUrls(Crawler $imagesCrawler)
         while ($iterator->valid()) {
             $srcsetAttribute = $iterator->current()->getAttribute('srcset');
             if ('' !== $srcsetAttribute) {
-                $srcset = array_map('trim', explode(',', $srcsetAttribute));
+                $pattern = "/(?:[^\"'\s]+\s*(?:\d+[wx])+)/";
```

Comment:
Could you add a comment to basically explain what it does?
I'm not regex fluent 🙂